cmd/local/conf/conf.go

```diff
@@ -167,15 +167,21 @@ func ReadConf(overrides map[string]string, configDir string) (*CollectConf, erro
 	}
 	simplelog.Debugf("logging parsed configuration from ddc.yaml")
 	defaultCaptureSeconds := 60
+	// set config struct
+	c := &CollectConf{}
 	// set node name
 	hostName, err := os.Hostname()
 	if err != nil {
 		hostName = fmt.Sprintf("unknown-%v", uuid.New())
 	}
+	//set default tmp dir (if its not in the conifg, use the OS default)
+	c.outputDir = GetString(confData, KeyTmpOutputDir)
+	if c.outputDir == "" {
+		c.outputDir = filepath.Join(os.TempDir(), getOutputDir(time.Now()))
```

Comment:
ok the thing negative with this is, this means that the "standard directory name structure" of ddc gets lost if the output dir is set, when really we just need to not use the temp dir when the output dir is set